### PR TITLE
Updates the WordPressKit pod version.

### LINKED ISSUE #15332: EXC_BREAKPOINT: Exception 6, Code 22148160, Subcode 8
**Sentry Url**: https://sentry.io/share/issue/c0b7ec4f4e044ae0a22a434200c0fa94/
**User Count**: 5
**Count**: 7
**First Release**: 16.0.0.6
**First Seen**: 2020-11-09T15:25:33Z
**Last Seen**: 2020-11-19T05:31:54Z
**24 Hours**: 1
**30 Days**: 7
**LogID:** EF48F337-EABF-4F6E-BB15-742F3CF664FD

```
EXC_BREAKPOINT: Exception 6, Code 22148160, Subcode 8
  File "PluginState.swift", line null, in value
  File "PluginState.swift", line 110, in PluginState.directoryURL.getter
  File "PluginViewModel.swift", line 335, in PluginViewModel.wpOrgLinkRow
  File "PluginViewModel.swift", line 31, in PluginViewModel.plugin.getter
  File "PluginViewModel.swift", line 423, in PluginViewModel.tableViewModel.getter
  File "PluginViewController.swift", line 87, in PluginViewController.bindViewModel
  File "PluginViewController.swift", line 77, in PluginViewController.viewDidLoad
  File "<compiler-generated>", line null, in PluginViewController.viewDidLoad
  File "main.swift", line 7, in main
...
(22 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by aerych**
Closes #15332

This change is already tested as part of https://github.com/wordpress-mobile/WordPressKit-iOS/pull/305

To test:
Make sure everything builds. 

Thanks again for the nudge @ScoutHarris !

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -43,7 +43,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 4.21.0'
+    pod 'WordPressKit', '~> 4.22-beta.2'
```
Comment (ScoutHarris) [inline Podfile]: We leave off the beta version so we don't have to update it _every_ time. So this should just be `4.22-beta`.

Comment (aerych) [reply Podfile]: That makes too much sense. 😆.

**Review by ScoutHarris — APPROVED** [review]
It's a go!

:shipit:

**Comment by aerych** [comment]
Yay! Thank you!

**Pull request merged**